.github/workflows/update-vrt.yml

```
@@ -0,0 +1,40 @@
+    runs-on: ubuntu-latest
+    if: |
+      contains(github.event.comment.body, '/update-vrt')
+    steps:
+      - name: Get PR branch
+        uses: xt0rted/pull-request-comment-branch@v2
+        id: comment-branch
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ steps.comment-branch.outputs.head_ref }}
+      - name: Set up environment
+        uses: ./.github/actions/setup
```

actionlint

description is required in metadata of "Setup" action at "/inmem/22/8cd9bdcd-3304-48b0-bfc2-9a529f0ab392/home/jailuser/git/.github/actions/setup/action.yml"

(action)